modules/govuk/manifests/apps/whitehall.pp

```diff
@@ -292,13 +293,9 @@
       # Restrict access to the healthcheck page. Grafana needs to be
       # able to access this from the monitoring machine. All other requests are blocked.
       # This is only useful for Carrenza, as AWS will not provide static IPs.
-      if $::aws_migration {
-        hidden_paths => [$health_check_path]
-      } else {
-        location $health_check_path {
-          allow $monitoring_ip
-          deny all;
-        }
+      location $health_check_path {
+        allow $monitoring_ip;
```

Comment:
To get this to interpolate correctly, I think [the string might have to be double quoted](https://puppet.com/docs/puppet/5.2/lang_variables.html#interpolation). And the current `$abc` variables will have to be escaped, something like `\$abc`.